### ISSUE TITLE
Linter not running; no error messages

### ISSUE THREAD

**Issue by brainsik** [jonlabelle/SublimeLinter-contrib-markdownlint#2]
name | version
------------ | -------------
macOS|10.13
SublimeText|3.0, Build 3143
SublimeLinter|3.10.1
SublimeLinter-contrib-markdownlint|1.0.3
markdownlint-cli|0.3.1
node|8.7.0

If I run the command-line tool by hand it will show linting errors. However, SublimeLinter keeps returning "No lint errors." no matter what sort of errors I introduce to the markdown file. I turned on SublimeLinter's "Debug Mode" and looked at the SublimeText console, but there are no error messages. I've restarted SublimeText many times.

Any thoughts on what else I can do to debug?

**Comment by jonlabelle** [comment]
Sounds like the linter, `markdownlint` isn't being picked by [SublimeLinter](http://www.sublimelinter.com/en/latest/usage.html#how-linter-executables-are-located).

Where do you have `markdownlint` (cli) installed?

**Comment by brainsik** [comment]
It's installed in `/usr/local/bin`.

**Comment by brainsik** [comment]
I'm pretty sure SublimeLinter is finding the executable. It shows `/usr/local/bin` in the "computer PATH" in the console:

```
SublimeLinter: computed PATH using /bin/bash:
/usr/local/bin
```

Also, this is the message one gets when it can not find the executable:

```
SublimeLinter: WARNING: shellcheck deactivated, cannot locate 'shellcheck' 
```

**Comment by jonlabelle** [comment]
Have tried troubleshooting steps in [Finding a linter executable](http://sublimelinter.readthedocs.io/en/latest/troubleshooting.html#finding-a-linter-executable)?

In the Sublime Text Console, with SublimeLinter Debug Mode enabled, what is the output of the command being run against the current Markdown file? For example, here's mine...

```
SublimeLinter: markdownlint: temp.md ['/usr/local/bin/markdownlint'] 
```

**My entire SublimeLinter Debug Output**

```
SublimeLinter: shellcheck linter loaded
SublimeLinter: markdownlint linter loaded
SublimeLinter: debug mode: on
SublimeLinter: temp directory: /var/folders/6k/4khc1vv52zbctbcxf3w9d4nc0000gn/T/SublimeLinter3-jon
SublimeLinter: user shell: /usr/local/bin/bash
SublimeLinter: computed PATH using /usr/local/bin/bash:
    /Users/jon/bin
    /usr/local/bin
    /usr/local/sbin
    /usr/bin
    /bin
    /usr/sbin
    /sbin
    /usr/local/share/dotnet
    /Library/Frameworks/Mono.framework/Versions/Current/Commands
    /usr/texbin
    /Applications/Wireshark.app/Contents/MacOS
    /Users/jon/bin
    /usr/local/bin
    /usr/local/sbin

SublimeLinter: markdownlint: temp.md ['/usr/local/bin/markdownlint']
SublimeLinter: markdownlint output:
/var/folders/6k/4khc1vv52zbctbcxf3w9d4nc0000gn/T/SublimeLinter3-jon/temp.md: 1: MD009 Trailing spaces
```

**Comment by brainsik** [comment]
Alright, I figured out what was happening. My markdown files weren't being seen as having the syntax type "Markdown", but were set as "Markdown Extended" since I use the [Markdown Extended package](https://packagecontrol.io/packages/Markdown%20Extended). When I switched things to "Markdown", the linter was found and started working.

Is there a way to tell the linter to work on additional syntax types?
